README.md

```diff
@@ -152,6 +152,12 @@ $ mpirun -np 16 -x LD_LIBRARY_PATH -mca btl_openib_receive_queues P,128,32:P,204
 
 Check your MPI documentation for arguments to the `mpirun` command on your system.
 
+## Keras
+
+Horovod supports Keras in a way very similar to regular TensorFlow. One notable requirement is to use
```

Comment:
Horovod supports Keras and regular TensorFlow in similar ways. 
Note: You must use `keras.optimizers.TFOptimizer` instead of native Keras optimizers.